src/data_base.cpp

```diff
@@ -62,7 +62,8 @@ data_base::data_base(const settings& settings, bool catalog)
   : closed_(true),
     catalog_(catalog),
     settings_(settings),
-    database::store(settings.directory, catalog, settings.flush_writes)
+    database::store(settings.directory, catalog,
```

Comment:
Seems like passing `catalog` but otherwise using `settings` is an inconsistency, but I haven't looked fully at the context.

Reply:
I'll match the pattern, though catalog is read from settings on the caller side.  Is the intention that no settings configuration is read by its direct user?

Reply:
Just pointing out the inconsistency. It may be better to only use settings.